care/emr/resources/observation/resource_spec.py

```
@@ -0,0 +1,96 @@
+    category: Coding | None = None
+    main_code: Coding | None = None
+    alternate_coding: CodeableConcept | None = None
+    subject_type: SubjectType
+    subject_id: UUID4
+    effective_datetime: datetime
+    performer: Performer | None = None
+    value_type: QuestionType
+    value: QuestionnaireSubmitResultValue
+    note: str | None = None
+    method: ValueSetBoundCoding[CARE_OBSERVATION_COLLECTION_METHOD.slug] | None = None
+    reference_range: list[ReferenceRange] = []
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)